packages/owl-bot-post-processor/src/core.ts

```diff
@@ -289,11 +296,69 @@ function getCloudBuildInstance() {
   return new CloudBuildClient();
 }
 
+const owlBotLockPath = '.github/.OwlBot.lock.yaml';
+export async function getOwlBotLock(
+  repoFull: string,
+  pull_number: number,
+  octokit: OctokitType
+): Promise<OwlBotLock> {
+  const [owner, repo] = repoFull.split('/');
+  const {data: prData} = await octokit.pulls.get({
+    owner,
+    repo,
+    pull_number,
+  });
+  const configRaw = (
+    await octokit.repos.getContent({
+      owner,
+      repo,
+      path: owlBotLockPath,
+      ref: prData.head.ref,
+    })
+  ).data as {content: string | undefined; encoding: string};
+  if (!configRaw.content) {
+    throw Error(`unable to find ${owlBotLockPath} in ${repoFull}`);
+  }
+  if (configRaw.encoding !== 'base64') {
```

Comment:
Playing with the API, I think the response for a file will always be `base64` encoded.